recipes/libvips/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+./configure --prefix="${PREFIX}" 
+make 
```

Comment:
Is it possible to compile it with multiple cores?
```suggestion
make -j${CPU_COUNT}
```